release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2436,6 +2434,12 @@ $ sed -i -e 's/registry-proxy.engineering.redhat.com/registry.redhat.io/g' \
     ./redhat-operator-index-manifests/imageContentSourcePolicy.yaml \
     ./redhat-operator-index-manifests/mapping.txt
 ----
++
+* For {product-title} on IBM Power Systems the following minimum requirements apply:
```

Comment:
Please mention powervm because this issue is specific to powervm.

Reply:
For {product-title} on IBM Power Systems on powervm, the following minimum requirements apply: